examples/list-fns.rs

```diff
@@ -3,70 +3,72 @@ use std::{env, error::Error, fs};
 use rnix::{types::*, NodeOrToken, SyntaxKind::*, SyntaxNode};
 
 fn main() -> Result<(), Box<dyn Error>> {
-    let file = match env::args().nth(1) {
-        Some(file) => file,
-        None => {
-            eprintln!("Usage: list-fns <file>");
-            return Ok(());
-        }
-    };
-    let content = fs::read_to_string(&file)?;
-    let ast = rnix::parse(&content).as_result()?;
-    let set = ast.root().inner().and_then(AttrSet::cast).ok_or("root isn't a set")?;
+    todo!()
+//     let file = match env::args().nth(1) {
```

Comment:
instead of commenting every line separately, it would be more appropriate to just use comment blocks.

Reply:
I'm not doing examples yet